cli/net.go

```diff
@@ -247,6 +290,51 @@ var NetConnect = &cli.Command{
 	},
 }
 
+func addrInfoFromArg(ctx context.Context, cctx *cli.Context) ([]peer.AddrInfo, error) {
```

Comment:
I know this is just copied over from existing code but this method is surprising.  It looks like it will either handle a slice of libp2p peer addr infos or it will take a single miner addr.  Either the name should carry over some of the complexity ("addrInfoFromAddrInfosOrMinerAddr") or you should add a comment describing what's going on.